extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketConnectorImpl.java

```diff
@@ -63,6 +66,12 @@ public Uni<WebSocketClientConnection> connect() {
             clientOptions.setMaxMessageSize(config.maxMessageSize().getAsInt());
         }
 
+        Optional<TlsConfiguration> tlsConfiguration = TlsConfiguration.from(tlsConfigurationRegistry,
```

Comment:
Shouldn't we do the same in the `BasicWebSocketConnectorImpl#connect()`?

Reply:
Ah yeah, I forgot to add it there - will do tomorrow

Reply:
Added